swagger/v1/swagger.json

```diff
@@ -5,6 +5,43 @@
     "version": "v1"
   },
   "paths": {
+    "/health": {
+      "get": {
+        "summary": "Gets system health",
+        "tags": [
+          "System"
+        ],
+        "produces": [
+          "text/plain"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully returned the system health"
+          }
+        }
+      }
+    },
+    "/poms/1,2,3": {
```

Comment:
Ah. Not sure I like this approach for :ids, it isn't really very explicit and doesn't feel right.

Reply:
Fair call ... 
Will take a look